ingestion/src/metadata/profiler/source/bigquery/profiler_source.py

```diff
@@ -59,3 +69,22 @@ def _copy_service_config(
                 )
 
         return config_copy
+
+    def create_profiler_interface(
+        self,
+        entity: Table,
+        table_config: Optional[TableConfig],
+    ) -> Union[SQAProfilerInterface, PandasProfilerInterface]:
+        """Create BigQuery profiler interface"""
+        profiler_interface: BigQueryProfilerSource = ProfilerProtocol.create(
```

Comment:
@duongphannamhung this needs to be `BigQueryProfilerInterface`. Once this is changed we can merge it